app/boards/nucleo_h745zi_q.overlay

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/gpio/gpio.h>
+
+/ {
+	chosen {
+		zephyr,canbus = &fdcan1;
+	};
+
+	cannectivity: cannectivity {
+		compatible = "cannectivity";
+		timestamp-counter = <&counter2>;
+
+		channel0 {
+			compatible = "cannectivity-channel";
+			can-controller = <&fdcan1>;
+			state-gpios = <&gpiob 0 GPIO_ACTIVE_HIGH>;
+			activity-gpios = <&gpioe 1 GPIO_ACTIVE_HIGH>;
```

Comment:
So all this PR is really adding is support for a separate activity LED on this board (which does not have a designated CAN activity LED)?  If that is all, I see no reason to pull it in, sorry.

Reply:
True - the changes are very small. But they are needed to make CANnectivity running.
On one hand without this changes one needs to do them each time the board shall run CANNectivity.
On the other hand the introduced board is still a development board without specific CANnectivity functionality, therefore supporting CANnectivity needs this overlay.

Reply:
If I understand you right the lines 14 to 22 are not needed and CANnectivity will work with this board just right from Zephyr main repository?

Reply:
> If I understand you right the lines 14 to 22 are not needed and CANnectivity will work with this board just right from Zephyr main repository?

None of the lines in this PR (except the bugfix for the missing `zephyr,canbus` chosen, which needs to go into Zephyr) are _needed_ for running CANnectivity on this board (have you tried)?

The addition of a separate activity LED is not really needed, as activity will be indicated on the state LED in it's absence. Separate activity LEDs are great for boards with dedicated CAN activity LEDs (e.g. boards intended for use as USB to CAN adapters). 

The only addition that _could_ warrant an overlay is really the added hardware timestamp counter, but I'm reluctant to start adding overlays for generic development boards only to add hardware timestamp support, a feature likely not many will require when using CANnectivity on a generic development board (or even know how to activate). These overlays will need to be maintained, tested - otherwise they will bitrot.

Reply:
Thank you.
On the other side I see problems with development boards in general - they are designed to cover as much applications as possible, therefore hardware conflicts are unavoidable. 
Sure, such conflicts may be solved by overlays, but the board is will still stay a development board and not a productive one.
I would suggest rejecting development board in general within CANnectivity.